entity-service-api/src/main/proto/org/hypertrace/entity/data/service/v1/entity_data_request.proto

```diff
@@ -100,3 +100,18 @@ message MergeAndUpsertEntityRequest {
 message MergeAndUpsertEntityResponse {
   Entity entity = 1;
 }
+
+message DeleteEntitiesRequest {
+  string entity_type = 1;
+  oneof delete_request {
+    StringList entity_ids = 2;
+    AttributeFilter filter = 3;
+  }
+}
+
+message StringList {
+  repeated string ids = 1;
+}
+
+message DeleteEntitiesResponse {
```

Comment:
How would we manage the response size? Based on filters it could mean deleting lots of entities. To return the ids we would need to fetch all the entity ids based on the filter (some pagination logic) and then return (need to consider the response size). What do you suggest?